ui/pages/login.tsx

```diff
@@ -299,13 +308,77 @@ const Login: NextPage = () => {
         text: "Invalid login method",
       });
     }
+    if (isSignInWithHardwareKey(node) && isSequencedLogin) {
+      node.meta.label.text = "Sign in using Passkey";
+    }
     return node;
   });
 
+  // automatically forward to single oidc provider if it is the only option
+  const isSingleOidcOption =
+    isSequencedLogin &&
+    renderFlow?.ui.nodes.length === 2 &&
+    renderFlow?.ui.nodes[1].group === "oidc" &&
+    (renderFlow?.ui.nodes[0].attributes as UiNodeInputAttributes).name ===
+      "csrf_token";
+  if (isSingleOidcOption) {
+    const csrfNode = renderFlow?.ui.nodes[0];
+    const oidcNode = renderFlow?.ui.nodes[1];
+    const oidcAttributes = oidcNode.attributes as UiNodeInputAttributes;
+    const csrfAttributes = csrfNode.attributes as UiNodeInputAttributes;
+    void handleSubmit({
+      method: "oidc",
+      provider: oidcAttributes.value as string,
+      csrf_token: csrfAttributes.value as string,
+    });
+  }
+
+  const isWebauthnAutologin = (): boolean => {
+    return (
+      localStorage.getItem(WEBAUTHN_AUTOLOGIN_KEY) === WEBAUTHN_AUTOLOGIN_VALUE
+    );
+  };
+
+  const toggleWebauthnSkip = () => {
+    if (isWebauthnAutologin()) {
+      localStorage.removeItem(WEBAUTHN_AUTOLOGIN_KEY);
+    } else {
+      localStorage.setItem(WEBAUTHN_AUTOLOGIN_KEY, WEBAUTHN_AUTOLOGIN_VALUE);
+    }
+  };
```

Comment:
I do appreaciate the clarity of these statements!
Since these are all based on constants, these functions should be outside of the component